vocab_files/attribute_concepts/soil-bulk-density-location.ttl

```diff
@@ -0,0 +1,20 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schema: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://linked.data.gov.au/def/nrm/1d7974b3-892b-47d5-8fdb-32f36f331e05>
+    a skos:Concept ;
+    dcterms:source """Laws M, McCallum K, Bignall J, O’Neill S, Sparrow B. (unpublished draft) ‘Soils module’ in ….. (eds) Ecological Field
+Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision
+making. Draft v 0.1 Report to DAWE. TERN, Adelaide""" ;
+    rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:definition "The sampling location/s on the plot where soil samples were collected for soil bulk density analysis." ;
+    skos:prefLabel "soil bulk density location" ;
```

Comment:
do we really need this concept, I am sure there is a "soil sample location"

Reply:
we have an attribute `soil sampling location description`